camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .detail(ResultErrorBuilder.attribute("enum.values"), result.getEnumChoices(entry.getKey()))
+                        .detail("enum.values", result.getEnumChoices(entry.getKey()))
```

Comment:
Maybe the easier solution is to keep all the standard stuffs uppercase with underscore (as it reflects most of the use cases in camel) and then you can eventually translate it on your side so we do not impose any "wire format" in camel